### ISSUE TITLE
Cursor Themeing support?

### ISSUE THREAD

**Issue by eylles** [themix-project/oomox#413]
is there any intention of adding support for theming cursors?
currently i have a working prototype but it is still rather crude
https://github.com/eylles/breeze-hacked-cursor-theme

**Comment by actionless** [comment]
nice stuff!
i think the easiest way would be to create just a separate plugin for that

You can take as an example existing plugins, like:
https://github.com/themix-project/oomox/tree/master/plugins/icons_gnomecolors
https://github.com/themix-project/oomox/tree/master/plugins/export_xresources

and let me know if you'll have any questions with them, i'll try my best to assist

**Comment by eylles** [comment]
Will make some changes to the design of the cursor as i don't like the arrows then i will look proper into the python script.

**Comment by actionless** [comment]
might be cool to leave old design as an option as well? (if that's not complicated to implement in the script)

**Comment by eylles** [comment]
Nah, it would just be matter of keeping the old svg file and adding another flag to the script to choose which svg file will be rendered.

**Comment by eylles** [comment]
hell how time flies... i have made no progress on this and i'm even considering taking the old numix cursor design to make it recolorable as that seems to have better contrast at least to me...

**Comment by actionless** [comment]
alright, good luck with it 😺👍
